draft-ietf-quic-ack-frequency.md

```diff
@@ -339,17 +333,32 @@ send an acknowledgement immediately on receiving a reordered ack-eliciting
 packet. This extension modifies this behavior.
 
 If the endpoint has not yet received an ACK_FREQUENCY frame, or if the most
-recent frame received from the peer has an `Ignore Order` value of `false`
-(0x00), the endpoint MUST immediately acknowledge any subsequent packets that
+recent frame received from the peer has a `Reordering Threshold` value that is
+not 0x00, the endpoint MUST immediately acknowledge any subsequent packets that
 are received out of order.
```

Comment:
```suggestion
When an endpoint receives an ACK_FREQUENCY frame with a `Reordering Threshold` value other than 0x00
the endpoint MUST immediately send an ACK frame when number of outstanding unacknowledged packets since
a reordering event was detected is larger then the `Reordering Threshold`.
```

Reply:
Sentence is wobbly, suggest more like:
/An endpoint that receives an ACK_FREQUENCY frame with a `Reordering Threshold` value other than 0x00, MUST immediately send an ACK frame when the number of outstanding unacknowledged packets since the last detected reordering event exceeds the `Reordering Threshold`./

Reply:
```suggestion
An endpoint, that receives an ACK_FREQUENCY frame with a Reordering
Threshold value other than 0x00, MUST immediately send an ACK frame
when the number of outstanding unacknowledged packets since
the last detected reordering event exceeds the Reordering Threshold.
```

Reply:
Btw. we might maybe want to discuss if this is really a MUST or just a SHOULD because it's at the end just a request that can't be enforced by the requester.

Reply:
I still think MUST, it's important for proper expected functioning. The word "immediately" appears in several IETF specs, and the sentiment is also that this needs to be done as soon as practically. The real sentiment is that this MUST NOT delay transmission of the response to the request.